### PR TITLE
DBZ-1220 FLOAT supports also single precision constraint

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1220

### CODE REVIEW

**Comment by jpechane** [comment]
Upstream issue https://github.com/antlr/grammars-v4/pull/1385

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java [hunk 1]
```diff
@@ -48,6 +48,33 @@ public void shouldGetExceptionOnParseAlterStatementsWithoutCreate() {
...
+        assertThat(table.columns().size()).isEqualTo(4);
+
+        final Column f1 = table.columnWithName("f1");
+        assertThat(f1.typeName()).isEqualTo("FLOAT");
+        assertThat(f1.length()).isEqualTo(-1);
+        assertThat(f1.scale().isPresent()).isFalse();
+
+        final Column f2 = table.columnWithName("f2");
+        assertThat(f2.typeName()).isEqualTo("FLOAT");
+        assertThat(f2.length()).isEqualTo(4);
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: Does the "precision" from FLOAT(p) really equate to the length (from FLOAT(L, D))? Reading the docs it seems to be rather a width in bits? I believe we dealt with this conversion before somewhere. But I might mix it up. Would be worth double-checking.

Comment (jpechane) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: @gunnarmorling It is parsed to length, but it is not important  for us as `FLOAT` is always converted to `double`.

Comment (gunnarmorling) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: What I meant is, won't the "length" value we optionally export as a header property be too large?

Comment (gunnarmorling) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: @jpechane, I now remembered were we had the similar case before: https://github.com/debezium/debezium-incubator/blob/master/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/antlr/listener/ColumnDefinitionParserListener.java#L153-L169

We set the length to 63 in this case (i.e. oversized). Probably should do the same here?

Comment (gunnarmorling) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: Discussed over chat that the `Column` length actually doesn't take any relevance for the Kafka data schema for a `float64` type. The value is only exported optionally as schema parameter, where it makes sense to reuse the existing length header field. I.e. all good.

**Pull request merged**